jenkins/builder.sh

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
-# Copyright 2016 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-
-set -o errexit
-set -o nounset
-set -o pipefail
-set -o xtrace
-
-# Find test scripts.
-script="$(dirname "${0}")/bootstrap.py"
-
-"${script}" --job="${1%.sh}" --repo=kubernetes/kubernetes --pull="${PULL_REFS:-}"
```

Comment:
I couldn't find anything using this @fejta @spxtr (?)

Reply:
fejta responded on slack, doesn't remember this file at all and thinks it's probably just a "zombie" so deleting it should be fine.